.github/labeler.yml

```diff
@@ -0,0 +1,25 @@
+design:
+  - 'napari/_qt/qt_resources/**/*'
+
+documentation:
+  - 'docs/**/*'
+  - 'docs/*'
+
+preferences:
+  - 'napari/_qt/**/*/preferences_dialog.py'
+  - 'napari/utils/settings/**/*.py'
+
+qt:
```

Comment:
Does this mean that if files in these locations get touched a PR will automatically get labeled with this name? That's pretty cool, I didn't know about that tool, is it a standard github action? Can you maybe add a link to it somewhere in a comment or at least in this PR. I'm a little worried this might start giving lots of labels to lots of things, but we can adjust if we don't like it

Reply:
>is it a standard github action?

Yes it is. Added some links as comments.